packages/terra-form-select/src/shared/_Option.jsx

```diff
@@ -94,6 +94,12 @@ const Option = ({
     }
   }
 
+  let label = display;
+  if (customProps.index && customProps.totalOptions) {
+    label = `${display} (${customProps.index} of ${customProps.totalOptions})`;
```

Comment:
```suggestion
    label = `${display} (${customProps.index} of ${customProps.totalOptions})`;
```
did we missed to get translations for string "of" here

Reply:
we have the translation for the string "of". Will update this to use it.

Reply:
Updated in https://github.com/cerner/terra-core/pull/3707/commits/33798ba649bf547f477821356fb9f65d9f824385